client/my-sites/checkout/composite-checkout/hooks/use-create-payment-methods/index.ts

```diff
@@ -395,7 +496,6 @@ export default function useCreatePaymentMethods( {
 
 	const existingCardMethods = useCreateExistingCards( {
 		storedCards,
-		stripeConfiguration,
```

Comment:
This argument is not used, so I'm removing it here. One of the benefits of type safety.